fraim/cli.py

```diff
@@ -227,9 +228,9 @@ def workflow_options_to_cli_args(options_class: type[Any]) -> dict[str, dict[str
             arg_config["type"] = int
         elif actual_type == float:
             arg_config["type"] = float
-        elif get_origin(actual_type) is list:
+        elif origin is list:
             arg_config["nargs"] = "+"
-        elif get_origin(actual_type) is Union or get_origin(actual_type) is UnionType:
+        elif origin in {Union, UnionType}:
```

Comment:
This fixes support for the newer Tuple syntax, so we can use `str | None` instead of `Optional[str]`.

Before if we used `str | None` arguments that should be lists were treated as strings.